client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/hooks/use-form-error-mapping.ts

```diff
@@ -96,10 +93,7 @@ export const useFormErrorMapping = (
 				};
 			}
 
-			if (
-				isEnglishLocale &&
-				code === 'automated_migration_tools_login_and_get_cookies_test_failed'
-			) {
+			if ( code === 'automated_migration_tools_login_and_get_cookies_test_failed' ) {
```

Comment:
Super nitpicky, but we should reverse this conditional so the constant is on the left-hand side ("Yoda" conditions as per WP coding standards)

Reply:
Thanks @sixhours.
I just double-checked if we are adopting the WP pattern, and I saw we just have 
164 instances of the Yoda style
1494 instances of the regular style.

I run a search using the following patterns:
Regular style
```
=== '
```
Yoda style

```
' === 
```

The javascript WordPress code standards also contain examples of "regular" style usage.
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/

So, I think the Yoda style is more applicable to PHP